rust/src/db.rs

```diff
@@ -21,6 +30,15 @@ pub type SqliteConnection = PoolConnection<Sqlite>;
 /// Wallet has to be managed by Rust as generics are not support by frb
 static DB: Storage<Mutex<SqlitePool>> = Storage::new();
 
+fn into_array(vec: Vec<u8>) -> [u8; 32] {
+    let slice = vec.as_slice();
+    let array: [u8; 32] = match slice.try_into() {
+        Ok(arr) => arr,
+        Err(_) => panic!("Expected a Vec of length {} but it was {}", 32, vec.len()),
```

Comment:
Why `panic` here and not use a `try_from` or similar?

Reply:
for me, this is not an error that can happen at runtime - provided that the attached unit test passes, this error condition is weeded out. 
rust-lightning types have to have the slice of that length in them, this API cannot change.

Reply:
fair enough